### PR TITLE
Cloud Updates

### PR DESCRIPTION
**Pull request by lzimm**
adding ability to specify regions (pending AMI configuration on gatekeeper) as well as adding hooks to destroy instances when removing them from the redirector dropdown. (re: https://github.com/preludeorg/operator-support/issues/94)

### CODE REVIEW

**Review by privateducky — CHANGES_REQUESTED** [review]
The region makes me a bit nervous to support right now... can you pull this out and just make this about deleting a redirector?

**Comment by lzimm** [comment]
> The region makes me a bit nervous to support right now... can you pull this out and just make this about deleting a redirector?

yup, updated

File: plugins/cloud/Cloud.html [hunk 1]
```diff
@@ -172,7 +209,7 @@
             platform: $('#aws-platform').val(),
             callback: $('#select-redirector option:selected').val()+':'+SETTINGS.local.tcp_port,
             config: {
-                key_name: path.basename(config.aws.key_name),
+                key_name: config.aws.key_name.split('/').slice(-1)[0],
```
Comment (privateducky) [inline plugins/cloud/Cloud.html:212]: what's this change for..? seems like the odd man out

Comment (lzimm) [reply plugins/cloud/Cloud.html:212]: that's odd, i must've had some git issues that brought back some dead code

Comment (lzimm) [reply plugins/cloud/Cloud.html:212]: my bad

Comment (lzimm) [reply plugins/cloud/Cloud.html]: should be fixed now

**Pull request merged**